src/Codeception/Module/Db.php

```diff
@@ -628,8 +628,7 @@ public function _cleanup($databaseKey = null, $databaseConfig = null)
             );
         }
         try {
-            // don't clear database for empty dump
-            if (!isset($this->databasesSql[$databaseKey]) || !count($this->databasesSql[$databaseKey])) {
+            if (false === $this->shouldCleanup($databaseConfig, $databaseKey)) {
```

Comment:
Just use `!` to evaluate a boolean.
Casing of function name is inconsistent with declaration.

Reply:
Ah balls, good spot on the case.

My preference is to avoid `!` where possible but happy to change here.